scripts/validate-front-matter/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM quay.io/giantswarm/python:3.9-alpine3.13
+
+RUN apk --no-cache add yaml-dev build-base git
+
+RUN pip install --global-option='--with-libyaml' pyyaml
```

Comment:
Not a blocker, but might consider pinning the apk package and pyyaml versions here to avoid breakage

Reply:
The Dockerfile is gone. We are using a pre-built image now, defined by https://github.com/giantswarm/docs-scriptrunner